.chloggen/service-remove-servicetelemetry.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I did a Github search: https://github.com/search?q=+language%3AGo+%2F%5B%28%2C+%5Cn%5Dextensions%5C.New%5B+%28%29%2C%5Cn%5D%2F+-repo%3Aopen-telemetry%2Fopentelemetry-collector+-repo%3Aopen-telemetry%2Fopentelemetry-collector-contrib+NOT+is%3Afork&type=code

I have not been able to find any public references to this symbol being used as an argument to a function, so I think it's okay to do in one go